src/main/java/com/spotinst/sdkjava/model/api/ocean/ecs/ApiImportClusterComputeConfiguration.java

```diff
@@ -0,0 +1,88 @@
+package com.spotinst.sdkjava.model.api.ocean.ecs;
+
+import com.fasterxml.jackson.annotation.JsonFilter;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.spotinst.sdkjava.client.rest.IPartialUpdateEntity;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonFilter("PartialUpdateEntityFilter")
+public class ApiImportClusterComputeConfiguration implements IPartialUpdateEntity {
+    @JsonIgnore
+    private Set<String>                         isSet;
+    private List<String>                        subnetIds;
+    private ApiImportClusterLaunchSpecification launchSpecification;
```

Comment:
Rename to ApiImportClusterResponseLaunchSpecification

Reply:
fixed